sdk/hatchery/hatchery.go

```diff
@@ -595,8 +595,8 @@ func SendSpawnInfo(ctx context.Context, h Interface, jobID int64, spawnMsg sdk.S
 
 func logStepInfo(ctx context.Context, step string, queued time.Time) {
 	if id := ctx.Value(LogFieldJobID); id != nil {
-		ctx = context.WithValue(ctx, LogFieldJobStep, step)
-		ctx = context.WithValue(ctx, LogFieldJobStepDelay, time.Since(queued).Milliseconds())
+		ctx = context.WithValue(ctx, LogFieldStep, step)
+		ctx = context.WithValue(ctx, LogFieldStepDelay, time.Since(queued).Nanoseconds())
```

Comment:
why do you want nano seconds, IMO it's not super easy to read as a humain

Reply:
To be coherent with "latency" field.